style/popup.less

```diff
@@ -1,71 +1,156 @@
 .wp-popup {
 	box-sizing: border-box;
 	padding: 0;
-	position: absolute;
-	border: solid rgba( 0, 0, 0, 0.05 ) 1px;
-	border-radius: 2px;
+	border-radius: 8px 8px 0 0;
 	background-color: white;
 	box-shadow: 0 30px 90px -20px rgba( 0, 0, 0, 0.3 ), 0 0 1px 1px rgba( 0, 0, 0, 0.05 );
-	height: 200px;
-	width: auto;
 	z-index: 110;
+	overflow: hidden;
+	position: absolute;
 
-	* {
-		box-sizing: border-box;
-	}
-
-	.wp-article-preview {
+	.wikipediapreviews {
```

Comment:
Do we plan to separate the popup CSS from the preview CSS in the future?

Reply:
ya, it's good to have those separately. I will do it in this PR.

Reply:
one thought of it! current CSS structure we have is `index` that includes `popup` (popup includes preview style)

my suggestion change:
no more `index.less` needed as `popup.less` (import in `popup.js`) `preview.less` (import in `preview.less`)

what do you think?

Reply:
Makes sense to me if it works well at runtime

Reply:
Sounds good to me